glue_ginga/plugins/Glue.py

```diff
@@ -58,9 +76,43 @@ def _data_removed_cb(self, msg):
         image = self.datasrc.remove(name)
         self.make_callback('data_out', image)
 
+    def _app_closed_cb(self, msg):
+        self.make_callback('app_closed', None)
+
     def get_data(self, name):
         return self.datasrc[name]
 
+    @staticmethod
+    def _data_to_image(data):
+        ids = data.component_ids()
+        data_np = data[ids[0]]
+        data_meta = {}
+
+        if hasattr(data.coords, 'header'):
+            h = AstroImage.AstroHeader()
+            h.update(data.coords.header)
+            data_meta['header'] = h
+
+        image = AstroImage.AstroImage(data_np=data_np, metadata=data_meta)
```

Comment:
We might need to delve a little more into the Glue class that underlies `DataImage`.  In the current use case, Glue is making a glue-like Data object so that Ginga can grok it.  I might be mistaken thinking that it could also be used for the reverse use case, although that seems like it should be possible.